### ISSUE TITLE
Integrate LuaJIT test suite

### ISSUE THREAD

**Issue by Totktonada** [tarantool/tarantool#4064]
https://github.com/LuaJIT/LuaJIT-test-cleanup/ or https://github.com/openresty/luajit2-test-suite

**Comment by Totktonada** [comment]
More test suites were integrated by [luavela](https://github.com/iponweb/luavela/). Maybe it worth to consider them too.

**Comment by ligurio** [comment]
@igormunkin Igor, was it done in scope of #4655?

**Comment by ligurio** [comment]
@igormunkin Igor, was it done in scope of #4655?

**Comment by igormunkin** [comment]
@ligurio, no, it wasn't. @avtikhon just adjusted the existing tests in [tarantool/luajit](https://github.com/tarantool/luajit) repo considering our Lua style guide and several practices mentioned by @Totktonada (see #4665 for more info and changes related to the issue).

**Comment by igormunkin** [comment]
The suite is introduced in scope of tarantool/luajit@d232a3f and the issue is resolved in scope of tarantool/luajit@75e6e90. The submodule was updated in tarantool in 2.8.0-108-g1458a2b27 (1458a2b), 2.7.1-97-ga97e40e3f (a97e40e), 2.6.2-94-g3d2a88fdc (3d2a88f), 1.10.9-52-gb30c11293 (b30c112).
